### DISCUSSION TITLE
Please don't merge a release commit with a copyright year update

### DISCUSSION THREAD

**Discussion by Zeblote** [NVIDIA-Omniverse/PhysX#100] [General]
This is extremely frustrating - now there are thousands of files with meaningless changes in the same commit as actual updates, making it really hard to tell what actually changed.

Any chance of someone re-pushing the last update as two separate commits?

**Comment by preist-nvidia** [comment]
Hi @Zeblote - we can't fix it for this release, unfortunately, but I'll keep this in mind for a future copyright year update.
